internal/commands/crypto.go

```diff
@@ -53,6 +54,9 @@ func newCryptoRandCmd(ctx *CmdCtx) (cmd *cobra.Command) {
 	cmd.Flags().StringP(cmdFlagNameCharSet, "x", cmdFlagValueCharSet, cmdFlagUsageCharset)
 	cmd.Flags().String(cmdFlagNameCharacters, "", cmdFlagUsageCharacters)
 	cmd.Flags().IntP(cmdFlagNameLength, "n", 72, cmdFlagUsageLength)
+	cmd.Flags().BoolP(cmdFlagNameQuiet, "q", false, cmdFlagUsageQuiet)
+	cmd.Flags().StringP(cmdFlagNameOutputFile, "o", "", cmdFlagUsageOutputFile)
+	cmd.Flags().StringP(cmdFlagNameDirectory, "d", "", cmdFlagUsageDirectory)
```

Comment:
I think having the file name and output directory as two separate flags in the same command could be confusing? The output directory or the full file path should be specified in the output file flag.

Just check out the behavior of the following commands:
```bash
authelia storage user totp export --file export.yaml
authelia storage user totp export --file nonexistant/export.yaml
mkdir existant && authelia storage user totp export --file existant/export.yaml
```